purchase_batch_invoicing/wizards/purchase_batch_invoicing_view.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+<record id="wizard_purchase_batch_invoicing_view" model="ir.ui.view">
+    <field name="name">Purchase Batch Invoicing</field>
+    <field name="model">purchase.batch_invoicing</field>
+    <field name="arch" type="xml">
+        <form>
+            <sheet>
+                <group attrs="{'invisible': [('purchase_order_ids', '=', False)]}">
+                    <field name="purchase_order_ids"/>
+                    <field name="grouping"/>
+                </group>
+                <div attrs="{'invisible': [('purchase_order_ids', '!=', False)]}">
+                    <p>No ready to invoice purchase orders selected.</p>
```

Comment:
idem
I am OK with the method but wouldnot it be better to check the condition before and raise a simple warning?